app/services/hyrax/workflow/workflow_importer.rb

```diff
@@ -1,27 +1,41 @@
 module Hyrax
   module Workflow
     class WorkflowImporter
-      # @api public
-      #
-      # Load all the workflows in config/workflows/*.json
-      # @return [TrueClass]
-      def self.load_workflows
-        Dir.glob(Rails.root + "config/workflows/*.json") do |config|
-          Rails.logger.info "Loading workflow: #{config}"
-          generate_from_json_file(path: config)
+      class << self
```

Comment:
@jeremyf I'll move it back.  My thought was that was the prefered syntax when you are creating many class methods.